telegram/ext/conversationhandler.py

```diff
@@ -168,6 +173,11 @@ class ConversationHandler(Handler[Update]):
         map_to_parent (Dict[:obj:`object`, :obj:`object`]): Optional. A :obj:`dict` that can be
             used to instruct a nested conversationhandler to transition into a mapped state on
             its parent conversationhandler in place of a specified nested state.
+        run_async (:obj:`bool`): Pass :obj:`True` to override the
+            :attr:`Handler.run_async` setting of all handlers (in :attr:`entry_points`,
+            :attr:`states` and :attr:`fallbacks`.
```

Comment:
```suggestion
            :attr:`states` and :attr:`fallbacks`).
```

look at me ~go~ closing parentheses